sklearn/inspection/__init__.py

```diff
@@ -12,14 +12,20 @@
     warnings.simplefilter("ignore", category=FutureWarning)
     from .partial_dependence import partial_dependence
 
+from ._partial_dependence import individual_conditional_expectation  # noqa
```

Comment:
I think that we don't need to have `#noqa` since they are added in `__all__`

Reply:
Thanks, I have removed the #noqa annotations.

Reply:
Removing these annotations attract the linting error 'E402 module level import not at top of file'. Reverting...